test/powershell/Language/Scripting/MyInvocation.Tests.ps1

```diff
@@ -0,0 +1,50 @@
+﻿Describe 'Testing of MyInvocation' -Tags "CI" {
+    It 'Script file' {
+
+        $MyInvocation.Line.ToLower().IndexOf('$null = & $scriptblock @parameters') | Should BeGreaterThan -1
```

Comment:
removed this test.

---
In reply to: [78846400](https://github.com/Powershell/Powershell/pull/2118#discussion_r78846400) [](ancestors = 78846400)